packages/perseus/src/widgets/__tests__/definition_test.js

```diff
@@ -54,70 +56,67 @@ describe("Definition widget", () => {
         // Act
         const definitionAnchor = screen.getByText("the Pequots");
         userEvent.hover(definitionAnchor);
-        jest.advanceTimersByTime(250);
 
         // Assert
         expect(container).toMatchSnapshot("open state");
     });
 
-    it("should display the definition on hover", () => {
+    it("should display the definition on hover", async () => {
         // Arrange
         renderQuestion(question);
 
         // Act
         const definitionAnchor = screen.getByText("the Pequots");
         userEvent.hover(definitionAnchor);
-        jest.advanceTimersByTime(250);
 
         // Assert
-        const tooltip = screen.getByRole("tooltip");
+        const tooltip = await screen.findByRole("tooltip");
         expect(tooltip).toBeVisible();
         expect(tooltip).toHaveTextContent("Definition text");
     });
 
-    it("should display the definition on click", () => {
+    it("should display the definition on click", async () => {
         // Arrange
         renderQuestion(question);
 
         // Act
         const definitionAnchor = screen.getByText("the Pequots");
         userEvent.click(definitionAnchor);
-        const tooltip = screen.getByRole("tooltip");
+        const tooltip = await screen.findByRole("tooltip");
 
         // Assert
         expect(tooltip).toBeVisible();
         expect(tooltip).toHaveTextContent("Definition text");
     });
 
-    it("should show via focus by the tab key", () => {
+    it("should show via focus by the tab key", async () => {
         // Arrange
         renderQuestion(question);
 
         // Act - Tab in to set focus
         userEvent.tab();
-        jest.advanceTimersByTime(250);
 
         // Assert
-        const tooltip = screen.getByRole("tooltip");
+        const tooltip = await screen.findByRole("tooltip");
         expect(tooltip).toBeVisible();
         expect(tooltip).toHaveTextContent("Definition text");
     });
 
-    it("should hide and blur by the tab key", () => {
+    it("should hide and blur by the tab key", async () => {
         // Arrange
         renderQuestion(question);
 
         // Act - Tab in to set focus, tab out to blur
         userEvent.tab();
-        jest.advanceTimersByTime(250);
+        await screen.findByRole("tooltip");
```

Comment:
Oh. TIL that `findBy...` waits up to second for the element to appear. That's useful!

Reply:
The time limit can be configured as well if need be.